Dockerfile

```diff
@@ -50,4 +56,10 @@ RUN make installcheck
 ###
 # this creates a usable postgres image but without the packages needed to build
 FROM base AS output
+
+# Automatically enable pg_duckdb
+RUN echo "shared_preload_libraries='pg_duckdb'" >> /usr/share/postgresql/postgresql.conf.sample
```

Comment:
ah, I didnt think that you could make changes to the _sample file_ (you know, it being named "sample" and all) and it would get picked up somehow later. TIL. :joy: I think it's still nice to be able to inject a config file for other purposes (like motherduck token) via docker-compose, but if this means that the image works without doing so, then great! 👍